src/elbwalker.ts

```diff
@@ -46,8 +47,9 @@ elbwalker.go = function (config: Elbwalker.Config = {}) {
     // custom: use the elbLayer
   }
 
-  // Register all handlers
-  initHandler();
+  if (config.prefix) {
```

Comment:
by setting a default parameter it'll alway be set, but I guess TS isn't happy here.
Also initHandler should always be called to work properly.
Try sth like `const prefix = config.prefix || 'elb'`